components/renku_data_services/app_config/config.py

```diff
@@ -9,120 +9,43 @@
-    K8sSchedulingClient,
-)
-from renku_data_services.k8s.config import KubeConfigEnv
-from renku_data_services.k8s.quota import QuotaRepository
-from renku_data_services.k8s_watcher import K8sDbCache
-from renku_data_services.message_queue.config import RedisConfig
-from renku_data_services.message_queue.db import EventRepository, ReprovisioningRepository
-from renku_data_services.message_queue.interface import IMessageQueue
-from renku_data_services.message_queue.redis_queue import RedisQueue
-from renku_data_services.metrics.core import StagingMetricsService
-from renku_data_services.metrics.db import MetricsRepository
-from renku_data_services.namespace.db import GroupRepository
-from renku_data_services.notebooks.config import NotebooksConfig, get_clusters
-from renku_data_services.notebooks.constants import AMALTHEA_SESSION_GVK, JUPYTER_SESSION_GVK
-from renku_data_services.platform.db import PlatformRepository
-from renku_data_services.project.db import (
-    ProjectMemberRepository,
-    ProjectMigrationRepository,
-    ProjectRepository,
-    ProjectSessionSecretRepository,
-)
-from renku_data_services.repositories.db import GitRepositoriesRepository
-from renku_data_services.search.db import SearchUpdatesRepo
-from renku_data_services.search.reprovision import SearchReprovision
-from renku_data_services.secrets.db import LowLevelUserSecretsRepo, UserSecretsRepo
-from renku_data_services.session import crs as session_crs
-from renku_data_services.session.constants import BUILD_RUN_GVK, TASK_RUN_GVK
-from renku_data_services.session.db import SessionRepository
-from renku_data_services.session.k8s_client import ShipwrightClient
-from renku_data_services.solr.solr_client import SolrClientConfig
-from renku_data_services.storage.db import StorageRepository
-from renku_data_services.users.config import UserPreferencesConfig
-from renku_data_services.users.db import UserPreferencesRepository
-from renku_data_services.users.db import UserRepo as KcUserRepo
-from renku_data_services.users.dummy_kc_api import DummyKeycloakAPI
-from renku_data_services.users.kc_api import IKeycloakAPI, KeycloakAPI
-from renku_data_services.users.models import UnsavedUserInfo
-from renku_data_services.utils.core import merge_api_specs, oidc_discovery
 
-default_resource_pool = models.ResourcePool(
-    name="default",
-    classes=[
-        models.ResourceClass(
-            name="small",
-            cpu=0.5,
-            memory=1,
-            max_storage=20,
-            gpu=0,
-            default=True,
-        ),
-        models.ResourceClass(
-            name="large",
-            cpu=1.0,
-            memory=2,
-            max_storage=20,
-            gpu=0,
-            default=False,
-        ),
-    ],
-    quota=None,
-    public=True,
-    default=True,
-)
+
+@dataclass
+class KeycloakConfig:
+    """Configuration values for keycloak."""
+
+    keycloak_url: str
+    keycloak_realm: str
+    client_id: str
+    client_secret: str
```

Comment:
Also make sure to import `field` from dataclasses at the top.


```suggestion
    client_secret: str = field(repr=False)
```